### DISCUSSION TITLE
File Management module upload permission for registered user

### DISCUSSION THREAD

**Discussion by ced7** [oqtane/oqtane.framework#4814] [Q&A] [answered]
Hi,
Is it possible to use the File Management module to allow registered users (or any custom-role users) to upload files ?
It seems that the "Upload Files" button is only accessible for Host or Administrators users. I've tried to check all the persmission boxes possible at different levels (page, module, folders,...), but no success so far.
Does the File Management module has any purpose for non-administrator users ?
(For info: I'm using an out-of-the-box Oqtane v5.2.4 with Sqlite)

**Comment by sbwalker** [comment, marked as answer]
The current File Management module was only designed for use within the Admin Dashboard (ie. by Administrators or Hosts). It would need to be enhanced for it to be used by other types of users.
